apps/web/src/pages/Applications/ApplicationSkillsPage/ApplicationSkillsPage.stories.tsx

```diff
@@ -41,7 +44,13 @@ const hasExperiencesProps: ApplicationSkillsProps = {
     },
     pool: {
       ...fakePoolCandidate.pool,
-      essentialSkills: [...experienceSkills],
+      poolSkills: experienceSkills.map((skill) => ({
+        id: faker.string.uuid(),
+        ...faker.helpers.arrayElement(
+          fakePoolCandidate?.pool?.poolSkills ?? [],
+        ),
```

Comment:
Are we not worried about adding the same skill multiple times here? I guess it doesn't really matter for a story?

Reply:
Yeah, I'm not sure. I just did my best to show as much as possible in the story. I personally don't care if there are duplicates but can change things if others would like to see unique skills here.

Reply:
Eh, seems fine, not worth the effort to change.